der/derive/src/sequence/field.rs

```diff
@@ -188,13 +192,29 @@ impl LowerFieldEncoder {
             })
         };
     }
+
+    /// Make this field context-specific.
+    fn apply_context_specific(&mut self, tag_number: &TagNumber, tag_mode: &TagMode) {
+        let encoder = &self.encoder;
+        let number_tokens = tag_number.to_tokens();
+        let mode_tokens = tag_mode.to_tokens();
+
+        self.encoder = quote! {
+            &::der::asn1::ContextSpecific {
```

Comment:
The `ContextSpecificRef` type should make the `Clone` unnecessary:

https://docs.rs/der/latest/der/asn1/struct.ContextSpecificRef.html